### PR TITLE
Migrate to CircleCI 2.0

### PR DESCRIPTION
**Pull request by abesto**

### CODE REVIEW

**Comment by abesto** [comment]
Huh, tests weren't triggered. Lemme go check the CircleCI config...

**Comment by abesto** [comment]
Right, “Build forked pull requests” was off. Enabled, pushing a noop commit to trigger a build

**Comment by abesto** [comment]
My current guess is that the Sleuth client also can't talk to Zipkin, because it's not available on `127.0.0.1`, trying to work around that...

**Comment by abesto** [comment]
Will run the frontend and backend in separate Docker containers configured to talk to a third Zipkin container. It's gonna be awesome.

**Comment by abesto** [comment]
It _is_ kind of awesome. Several yaks have been hurt in the making of this PR.

File: scripts/runAcceptanceTests.sh [hunk 1]
```diff
@@ -50,13 +47,17 @@ function curl_local_health_endpoint() {
...
+    docker_exec frontend curl --fail -m 5 "127.0.0.1:8081" || return 1
+    docker_exec frontend curl --fail -m 5 "127.0.0.1:8081" || return 1
+    echo -e "\n\nSuccessfully sent two test requests!!!"
 }
 
 function run_docker() {
-    docker-compose -f "${ROOT}/docker/docker-compose.yml" kill || echo "Failed to kill any docker containers"
-    docker-compose -f "${ROOT}/docker/docker-compose.yml" pull
-    docker-compose -f "${ROOT}/docker/docker-compose.yml" up -d
+    our_docker_compose kill || echo "Failed to kill any docker containers"
```
Comment (codefromthecrypt) [inline scripts/runAcceptanceTests.sh:56]: hehe our_docker_compose

**Review by codefromthecrypt — APPROVED** [review]
thanks tons

**Comment by codefromthecrypt** [comment]
@marcingrzejszczak you likey?

**Review by marcingrzejszczak — APPROVED** [review]

**Comment by marcingrzejszczak** [comment]
Looks good!

**Pull request merged**